### ISSUE TITLE
ERR_UNSAFE_PORT when opening book

### ISSUE THREAD

**Issue by carman247** [Mantano/iridium#106]
Occasionally I get a blank WebView screen with the message ERR_UNSAFE_PORT

I think the IP and port it's trying to use are 127.0.0.1:4045

**Comment by jmgeffroy** [comment]
Hello @carman247, can you please try to deactivate the use HTTP server? This can be done by changing the default value here: https://github.com/Mantano/iridium/blob/fcc2e2e690276e2a1ce412d3c15ee813b7c13354/components/navigator/lib/src/publication/publication_controller.dart#L43-L43
It could be a good idea to search for places where this might be explicitly given a different value.
Please let me know if it works or not. I haven't tested on iOS for a while, and right now today, I can't.

**Comment by carman247** [comment]
Ok, will this affect performance at all? I'll test it and see if the problem continues to occur.

By the way, I was testing on an Android 13 device.

**Comment by jmgeffroy** [comment]
Not at all. In fact, we had implemented this optimization in our (unpublished) Mantano Reader app, which was written in Java/Kotlin, and it improved the performances very signicantly. With our Flutter/Dart implementation we expected similar gains, but we found no measurable difference, which is strange and should be investigated one of these days. However, we feel that for now the performances are good enough and we should focus on other stuff. 
Don't hesitate sharing your observations.
